src/RestApi/StoreApi/Schemas/CartSchema.php

```diff
@@ -160,22 +169,23 @@ protected function get_properties() {
 	public function get_item_response( $cart ) {
 		$cart_item_schema = new CartItemSchema();
 		return [
-			'currency'           => get_woocommerce_currency(),
-			'items'              => array_values( array_map( [ $cart_item_schema, 'get_item_response' ], array_filter( $cart->get_cart() ) ) ),
-			'items_count'        => $cart->get_cart_contents_count(),
-			'items_weight'       => wc_get_weight( $cart->get_cart_contents_weight(), 'g' ),
-			'needs_shipping'     => $cart->needs_shipping(),
-			'total_items'        => wc_format_decimal( $cart->get_subtotal() ),
-			'total_items_tax'    => wc_format_decimal( $cart->get_subtotal_tax() ),
-			'total_fees'         => wc_format_decimal( $cart->get_fee_total() ),
-			'total_fees_tax'     => wc_format_decimal( $cart->get_fee_tax() ),
-			'total_discount'     => wc_format_decimal( $cart->get_discount_total() ),
-			'total_discount_tax' => wc_format_decimal( $cart->get_discount_tax() ),
-			'total_shipping'     => wc_format_decimal( $cart->get_shipping_total() ),
-			'total_shipping_tax' => wc_format_decimal( $cart->get_shipping_tax() ),
-			'total_tax'          => wc_format_decimal( $cart->get_total_tax() ),
-			'total_price'        => wc_format_decimal( $cart->get_total() ),
-			'tax_lines'          => $this->get_tax_lines( $cart ),
+			'currency'            => get_woocommerce_currency(),
+			'currency_minor_unit' => wc_get_price_decimals(),
+			'items'               => array_values( array_map( [ $cart_item_schema, 'get_item_response' ], array_filter( $cart->get_cart() ) ) ),
+			'items_count'         => $cart->get_cart_contents_count(),
+			'items_weight'        => wc_get_weight( $cart->get_cart_contents_weight(), 'g' ),
+			'needs_shipping'      => $cart->needs_shipping(),
+			'total_items'         => ( new MoneyValue() )->from_decimal( $cart->get_subtotal(), wc_get_price_decimals() ),
```

Comment:
`wc_get_price_decimals` is the minor unit. Quoting the updated description:

> Currency minor unit (number of digits after the decimal separator) used for cart item prices.

So for USD it's default 2, but the user can customise this unit hence there is no hard relation to currency code in WooCommerce.

Reply:
> So for USD it's default 2, but the user can customise this unit hence there is no hard relation to currency code in WooCommerce.

Ahh so you're saying the money value might be in USD currency, but the site owner has customized that value so it's always calculated to a precision of whatever decimal places they have set?  If so, I get it, but then we shouldn't call this minor unit, because according to the links you've linked to, _minor unit_ IS definitely related to currency.  The minor unit of USD is not 4, not 5, it is 2.

If this is just an arbitrary decimal value that user's can set irregardless of the currency minor unit spec, then let's make that clear.

Reply:
Also, if user's can set the decimal value arbitrarily, does this mean that what is saved in the db could be to that precision or is what is saved in the db _always_ according to the currency?

Reply:
@nerrad The value is the DB uses whatever setting the user has chosen. That's probably one of the reasons why cents were never used for storage, because changes to that value, or the store currency, would invalidate the stored strings in the DB.

> we shouldn't call this minor unit, because according to the links you've linked to, minor unit IS definitely related to currency. The minor unit of USD is not 4, not 5, it is 2.

Isn't this getting pedantic? In majority of cases they are going to align, we just need to be cautious because an arbitrary value _could_ be used. In any case, as long as the API response returns the correct unit, there is no problem for the consumer. Additionally, regardless of naming, the consumer needs to use the same logic.

We should call it minor_unit in the API to avoid using our own custom terms. And who knows, maybe this core setting can "go away" in the future leaving us with the correctly named minor_unit in the API :)